etc/webapi.xml

```diff
@@ -68,11 +68,11 @@
     <route url="/V1/adyen/originKey" method="GET">
         <service class="Adyen\Payment\Api\AdyenOriginKeyInterface" method="getOriginKey"/>
```

Comment:
I think we can remove the `originKey` endpoint entirely by now.

Reply:
Indeed we are not using it

Reply:
Removed.